### DISCUSSION TITLE
Ignore comments in file input containing IDs per line

### DISCUSSION THREAD

**Discussion by aviadavi** [projectdiscovery/nuclei#5939] [Ideas]
Ability to add comment per line , for example:

```
tls-sni-proxy # This is example 1
some-other-id # This is example 2
```

for -exclude-id

https://github.com/projectdiscovery/nuclei/issues/1442

**Comment by GeorginaReeder** [comment]
Hi there @aviadavi !

Just to confirm, are you suggesting this as a feature? If so, please feel free to create a new feature request and we'll take a closer look into it. Thanks!

**Reply by aviadavi** [reply]
yes, will do. thanks,
